pkgs/servers/sql/postgresql/ext/plr.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub, pkg-config, R, postgresql }:
+
+stdenv.mkDerivation rec {
+  name = "plr";
+  src = fetchFromGitHub {
+    owner = "postgres-plr";
+    repo = "plr";
+    rev = "c82fe1984b51f53ce7a4ad6853815bc9554fa390";
+    sha256 = "1cnjkrxk9s14n7p25bv7zwibw8q379ajhqpik66rw80mwci44xka";
+  };
```

Comment:
```suggestion
  pname = "plr";
  version = "8.4.1";

  src = fetchFromGitHub {
    owner = "postgres-plr";
    repo = "plr";
    rev = "REL${builtins.replaceStrings ["."] [":"] version}";
    sha256 = "1cnjkrxk9s14n7p25bv7zwibw8q379ajhqpik66rw80mwci44xka";
  };
```

Reply:
@SuperSandro2000 updated the PR, now set to 8.4.1.